README.md

```diff
@@ -337,74 +337,6 @@ used refinement function. The built-in `completion-styles` support the
 * The option `selectrum-should-sort` controls whether preprocessing
   functions should sort.
 
-### Complementary extensions
```

Comment:
Why remove the complementary section completely? This is not a good change. I would at least link from the readme to Consult, Embark, Marginalia and Orderless. It is justified to have them mentioned prominently in the readme. This is better if someone searches the page and also for indexing.

Reply:
I have moved the section to the main wiki page instead.

Reply:
Mini-frame is also okay to be mentioned, but flimenu, browse-kill-ring and amx have been superseded and are not up-to-date. I am biased here obviously.

Reply:
Yes, I disagree with this change. It is better to mention the packages officially in the readme. At least the main ones, since this is an endorsement in contrast to mentioning it in the wiki. Note that the wiki is much less directly accessible and everyone can edit it.

Reply:
The idea is to have the README only deal with Selectrum but I have not hard preference on this, if you think the main wiki page isn't good enough I can also move it back.

Reply:
I would have added a prominent link to it in the usage section. I see it as a benefit that everyone can edit it, usually people not part of the project will behave responsible in my experience. But okay I will bring it back, at least in part.

Reply:
Yes, I hope you move it back. As I said, I thought before about removing flimenu, browse-kill-ring and amx. I am also okay to have them here since I don't want to make an overly biased impression. But given that we are collaborating relatively closely on the Selectrum, Consult, Embark, Orderless, Marginalia packages, it is good to have them mentioned here prominently.

Reply:
Note that many people seem to get a lot of value out of our package combination. It really does not make sense to me - you added prominent links to Helm and Ivy (which both do not link back here) and you removed the links to our own packages :confused:

Reply:
I will reduce it to those and link for more on the wiki then. Having them more prominent is probably beneficial, I agree.

Reply:
> Note that many people seem to get a lot of value out of our package combination. It really does not make sense to me - you added prominent links to Helm and Ivy (which both do not link back here) and you removed the links to our own packages

I agree with @minad FWIW. You shouldn't be shy of recommending an elegant seamless UX, even as you promote the decoupled nature of the projects.

Reply:
I believe the ones which should be mentioned are the following: Selectrum, Consult, Embark, Orderless, Marginalia and Prescient. See also https://github.com/minad/consult#recommended-packages.

Reply:
I value this packages myself this was just me trying to clean things up, my plan was to prominently link to the sample setups that are going to be added.

Reply:
Thanks for changing it back. Yes I understand, but it really makes a difference is something is linked on the wiki or directly. The difference is mainly for people who want to get a quick overview. The wiki is for people who dig deeper.

Reply:
Okay, I'm glad you prompted me for it!